README.md

```diff
@@ -5,13 +5,32 @@ Dependencies:
 - NodeJS
 - nginx
 - elasticsearch
-- python
 
-How to run?
------------
+- OR - 
```

Comment:
It's better to remove `-` before OR. You can see example [here](https://gist.github.com/xrandom/602306a44714235584a6).